rel/overlay/etc/default.ini

```diff
@@ -147,6 +147,27 @@ max_db_number_for_dbs_info_req = 100
 ; prevent non-admins from accessing /_all_dbs
 ; admin_only_all_dbs = true
 
+;;;; Moved from [httpd] ;;;;;
+authentication_handlers = {couch_httpd_auth, cookie_authentication_handler}, {couch_httpd_auth, default_authentication_handler}
```

Comment:
For the scheme of checking chttpd first then httpd to work, the settings which moved should all be commented out, so we should try to have this one commented out as well

Reply:
authentication_handlers moved back to the [httpd].